app/models/miq_ae_instance.rb

```diff
@@ -136,6 +136,12 @@ def self.get_homonymic_across_domains(user, fqname, enabled = nil)
     MiqAeDatastore.get_homonymic_across_domains(user, ::MiqAeInstance, fqname, enabled)
   end
 
+  def self.get_homonymic_across_domains_noprefix(user, path, enabled = nil)
+    return [] if path.blank?
```

Comment:
What if path is a fully qualified path name?

Reply:
It just parse the `path => 'domain/namespace1/namespace2/class/instance'` to `{ namespace => 'domain/namespace1/namespace2', klass => 'class', instance => 'instance'}` .... and in that case it mostly returns blank array ([]), because there is no such a namespace. It could return some object only if there will be another `miq_ae_instance` with `fqname` equal to `'some_domain/fqname_from_method_call'`. This is the only case I have in my mind.